### PR TITLE
Support Place and Organisation types

### PR DESCRIPTION
**Pull request by tomrf1**
Tested in CODE-ZEBRA, e.g. for `/tags?categories=organisation`
```
{
    "id": "technology/google",
    "type": "keyword",
    "sectionId": "technology",
    "sectionName": "Technology",
    "webTitle": "Google",
...
    "tagCategories": [
        "Organisation"
    ],
    "entityIds": [
        "organisation/google"
    ]
}
```

### CODE REVIEW

**Comment by shtukas** [comment]
:+1:

**Comment by mchv** [comment]
@tomrf1 do we want to be a bit more specific than place in the future?

File: src/main/scala/com/gu/entity_indexer/KnowledgeGraph.scala [hunk 1]
```diff
@@ -11,12 +12,18 @@ case class GoogleEntity(name: String, `@type`: Set[String], description: Option[
...
-    s"""https://kgsearch.googleapis.com/v1/entities:search?query=${term.replaceAll(" ","+")}&key=$key&limit=5&types=Person&types=Place&types=Organization"""
+  private def buildUrl(term: String, key: String, types: Set[String]): String = {
+    val typesParams = types.map(t => s"types=${americanise(t)}").mkString("&")
+    s"""https://kgsearch.googleapis.com/v1/entities:search?query=${term.replaceAll(" ","+")}&key=$key&limit=5&$typesParams"""
   }
 
-  def getEntities(term: String, key: String): Option[GoogleResponse] = {
-    val request = new Request.Builder().url(buildUrl(term, key)).build
+  def americanise(correct: String) = correct match {
+    case "Organisation" => "Organization"
```
Comment (mchv) [inline src/main/scala/com/gu/entity_indexer/KnowledgeGraph.scala:21]: 🤣

Comment (mchv) [reply src/main/scala/com/gu/entity_indexer/KnowledgeGraph.scala:21]: you should obviously have call this method `americanize `

**Comment by mchv** [comment]
👍

**Comment by tomrf1** [comment]
@mchv possibly. E.g. schema.org [defines](https://schema.org/City) `Place > AdministrativeArea > City`. I'm not sure how we'll handle the introduction of new subtypes.

File: src/main/scala/com/gu/entity_indexer/KnowledgeGraph.scala (hunk 1 above)
Comment (tomrf1) [reply src/main/scala/com/gu/entity_indexer/KnowledgeGraph.scala:21]: ಠ_ಠ

**Pull request merged**